### PR TITLE
[BH-1922] Add a Summary screen to Focuse timer

### PR DESCRIPTION
**Pull request by ghost**
Added a summary screen that tells the user how much time they spent in focus. This screen is displayed after the last focus time.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [ ] Has changelog entry added

### CODE REVIEW

File: products/BellHybrid/apps/application-bell-focus-timer/presenter/FocusTimerPresenter.cpp [hunk 1]
```diff
@@ -102,7 +113,22 @@ namespace app::focus
...
         betweenSessionTimer.stop();
-        app->switchWindow(focus::window::name::main);
+
+        const auto elapsed = std::chrono::duration_cast<std::chrono::minutes>(timer->getElapsed());
+        const auto minutesInFocus =
+            elapsed.count() + (allFocusSessionsCount - focusSessionsLeft) * focusSessionDuration.count();
+        const auto sumOfFocusTime =
+            std::to_string(minutesInFocus) + " " + utils::language::getCorrectMinutesAccusativeForm(minutesInFocus);
+        const auto textToComplete = utils::translate(app_focus_timer_summary);
+        const auto summaryText    = createSummaryText(textToComplete, sumOfFocusTime);
```
Comment (Maciej-Mudita) [inline products/BellHybrid/apps/application-bell-focus-timer/presenter/FocusTimerPresenter.cpp:123]: If I see correctly, the text "app_focus_timer_summary" is translated twice. The first time on line 122 and the second time inside the "createSummaryText" function

File: products/BellHybrid/apps/application-bell-focus-timer/presenter/FocusTimerPresenter.cpp [hunk 2]
```diff
@@ -15,6 +15,17 @@
 namespace
 {
     static constexpr std::chrono::milliseconds betweenSessionDelayTime{5000};
+    static constexpr auto summaryWindowTimeout = std::chrono::seconds{5};
+    static constexpr auto app_focus_timer_summary =
+        "<text>Well done!  <token>$VALUE</token> focus complete.</text>"; // TODO: move to translations
```
Comment (Maciej-Mudita) [inline products/BellHybrid/apps/application-bell-focus-timer/presenter/FocusTimerPresenter.cpp:20]: consider adding new text in JSON for languages - if there are no translations, leave the text empty or "-"

**Review by Maciej-Mudita — APPROVED** [review]

File: products/BellHybrid/apps/application-bell-focus-timer/presenter/FocusTimerPresenter.cpp [hunk 3]
```diff
@@ -15,6 +15,17 @@
 namespace
 {
     static constexpr std::chrono::milliseconds betweenSessionDelayTime{5000};
```
Comment (Lefucjusz) [inline products/BellHybrid/apps/application-bell-focus-timer/presenter/FocusTimerPresenter.cpp:17]:
```suggestion
    constexpr std::chrono::milliseconds betweenSessionDelayTime{5000};
```

No need for `static` keyword, all of these values are already in anonymous namespace which serves the same purpose

File: products/BellHybrid/apps/application-bell-focus-timer/presenter/FocusTimerPresenter.cpp [hunk 4]
```diff
@@ -15,6 +15,17 @@
...
+    static constexpr auto summaryWindowTimeout = std::chrono::seconds{5};
+    static constexpr auto app_focus_timer_summary =
+        "<text>Well done!  <token>$VALUE</token> focus complete.</text>"; // TODO: move to translations
+
+    std::string createSummaryText(const std::string &str, const std::string minutesOfFocus)
```
Comment (Lefucjusz) [inline products/BellHybrid/apps/application-bell-focus-timer/presenter/FocusTimerPresenter.cpp:22]:
```suggestion
    std::string createSummaryText(const std::string &str, const std::string &minutesOfFocus)
```

**Review by Lefucjusz — APPROVED** [review]

**Review by dawidwojtas-mudita — APPROVED** [review]

**Pull request merged**